.aicoe-ci.yaml

```diff
@@ -0,0 +1,5 @@
+check:
+  - thoth-build
+  - thoth-precommit
+release:
+  - upload-pypi-sesheta
```

Comment:
@harshad16 quick question how do I configure this component - like where it takes the credentials, which PyPi repo it accesses etc.?

Reply:
ah! we just need to assign @sesheta  as a maintainer in PyPI based upon the package declared in setup.py.

Reply:
done! :slightly_smiling_face: That was easy! :+1: